docs/reference/cli.md

```diff
@@ -193,6 +200,92 @@ time-component | 1     | Provides time-related functions
 - `--output-format <FORMAT>`: Output format (json, yaml, table) [default: json]
 - `--plugin-dir <PATH>`: Component storage directory
 
+## Registry Management
+
+The registry commands provide convenient access to components defined in the `component-registry.json` file. This centralized registry makes it easy to discover and fetch commonly used components without needing to remember their full OCI URIs.
```

Comment:
Don't need to mention component-registry.json because its internal implementation details.